index.html

```diff
@@ -4,9 +4,47 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="/styles/index.css" rel="stylesheet">
     <title>Weather Report</title>
 </head>
 <body>
-    
+    <header>
+        <a id = "sky">☀️☀️☀️☀️☀️☀️☁️☀️☀️☀️</a>
+    <section>
+        <select id = "skyChoice">
```

Comment:
Very tiny thing, but for HTML attributes, the convention is no space between assignment: `id="sky"`